.github/workflows/examples-k32w.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
         if: github.actor != 'restyled-io[bot]'
 
         container:
-            image: connectedhomeip/chip-build-k32w:0.4.22
+            image: connectedhomeip/chip-build-k32w:latest
```

Comment:
0.2.23? I don't really think we should be using latest here, as evidence by the fact that when I uploaded the container for you it broke our entire CI system for everyone.

Reply:
@mspang the changes introduced in this PR (specifically the below commit) should have solved the compilation issues. Basically the latest k32w docker image is enabled for MR3 of the K32W SDK.

commit 22ecca2fea0cd79f2cecd29ce8469f1a67b02355
Author: Doru Gucea <doru-cristian.gucea@nxp.com>
Date:   Mon Apr 19 06:36:48 2021 -0700

[K32W] Update K32W platform to MR3 of the SDK
    
Signed-off-by: Doru Gucea <doru-cristian.gucea@nxp.com>

Reply:
I understand, but the same can easily happen again and it violates my expectations. At the time new containers are pushed, they are untested by any of the GitHub workflows (we can fix that, but it will require building the containers inline in CI whenever the docker files change).

Reply:
Hm, ok. How should I proceed then?

Reply:
My position is it should say 0.4.23 but this is a broader issue no something K32W specific one.

Reply:
Now, it's clear, sorry for the confusion, changed latest to 0.4.23.